CompatBot/EventHandlers/EmpathySimulationHandler.cs

```diff
@@ -42,7 +42,8 @@ public static async Task OnMessageCreated(MessageCreateEventArgs args)
                 return;
 
             var similarList = queue.Where(msg => content.Equals(msg.Content, StringComparison.InvariantCultureIgnoreCase)).ToList();
-            if (similarList.Count > 2 && similarList.Select(msg => msg.Author.Id).Distinct().Count() > 2)
+            var rng = new Random();
+            if (similarList.Count > rng.Next(2, 5) && similarList.Select(msg => msg.Author.Id).Distinct().Count() > 2)
```

Comment:
you probably meant to check if unique author count is random, not that there were random number of messages with the same content; also if you create new random every time, just inline it